content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md

```diff
@@ -60,3 +60,8 @@ The Notebook UI of Machine Learning Workbench was a custom implementation create
   - Fixed the issue with training more epochs on the integrated Jupyter Notebook within the MLW by upgrading tensorflow to 2.0.
   - Fixed the issue with committing the project when an existing project commit task is in-progress.
   - Fixed the issue with deleting the project when any of the project related task is in-progress.
+
+
+* Fix release:
+  - A potential security issue was detected in MLW microservice. To mitigate this, a fix release for MLW 10.13 is provided.
+    This fix release limits the access of "Machine Learning User" and "Machine Learning Manager" user groups. Detailed changes are documented here: [10.13 fix release](/release-10-13-0/announcements-10-13-0/#:~:text=the%20Apama%20documentation.-,Machine%20learning%20Workbench,-Implemented)

```

Comment:
Detailed changes are documented here: [10.13 fix release] > Detailed changes are documented under [Important announcements].
The link doesn´t look correct, should just be /release-10-13-0/announcements-10-13-0/

Reply:
I have updated the link